docs/editor_manual/finding_your_way_around/the_explorer_page.md

```diff
@@ -4,31 +4,22 @@ The Explorer page allows you to view a page's children and perform actions on th
 
 ![](../../_static/images/screen05_explorer_page.png)
 
--   The name of the section you are looking at is displayed below the breadcrumb (the row of page names beginning with the home icon). Each section is also itself a page (in this case the homepage). Clicking the title of the section takes you to the Edit screen for the section page.
+-   The name of the section you are looking at is displayed at the top. Each section is also itself a page (in this case the homepage). Clicking the title of the section takes you to the Edit screen for the section page.
 -   As the heading suggests, below are the child pages of the section. Clicking the titles of each child page will take you to its Edit screen.
-
-![](../../_static/images/screen06_explorer_page_arrow.png)
-
 -   Clicking the arrows will display a further level of child pages.
 
 ![](../../_static/images/screen07_explorer_page_breadcrumb.png)
 
--   As you drill down through the site the breadcrumb (the row of pages beginning with the home icon) will display the path you have taken. Clicking on the page titles in the breadcrumb will take you to the Explorer screen for that page.
-
-![](../../_static/images/screen08_add_page_button.png)
-
--   To add further child pages press the Add child page button below the parent page title. You can view the parent page on the live site by pressing the View live button. The Edit button will take you to the page edit screen.
+-   As you drill down through the site, the breadcrumb (the row of pages beginning with the home icon) will display the path you have taken. Clicking on the page titles in the breadcrumb will take you to the Explorer screen for that page.
 
 ![](../../_static/images/screen09.1_page_more_dropdown.png)
 
--   Clicking on More dropdown will show a list of actions for the parent page, like Move, Copy, Delete, Unpublish, History. There will also be a Sort menu order button that will take you to the ordering page (see below)
+-   Clicking on the Actions dropdown will show a list of actions for the parent page, like Move, Copy, Delete, Unpublish, History. There will also be a Sort menu order button that will take you to the ordering page (see below)
+-   To add further child pages press the Add child page button. You can view the parent page on the live site by pressing the View live button. The Edit button will take you to the page edit screen.
 
 ![](../../_static/images/screen09.2_bulk_action_checkboxes.png)
 
 -   There is a checkbox on the left of each page row, hovering over the row will make it visible. There is also a checkbox at the top left of the header, used to select or deselect all other checkboxes.
-
-![](../../_static/images/screen09.3_bulk_action_bar.png)
```

Comment:
Removed because I thought this would be easier to understand by showing the bar directly in the above screenshot.